### DISCUSSION TITLE
Integration with SDAC

### DISCUSSION THREAD

**Discussion by Mrtools1998** [AliDehbansiahkarbon/EasyDBMigrator#5] [Q&A]
Is it possible to integrate with SDAC?

**Comment by AliDehbansiahkarbon** [comment]
Certainly, but it takes a bit of time and energy.
There is a connection manager to control each database type (SQL Server, MySQL, ...) operations let say your target DB is MySQL, in this case 
you need to define a new connection manager like "TMySQLConnection" (defined in the unit "EasyDB.ConnectionManager.MySQL") and it must be inherited from "TConnection" class which is defined in the unit "EasyDB.ConnectionManager.Base".

In this new ConnectionManager you can use Any other data controller object ineasted of FireDAC.
